docs/usage/README.md

```diff
@@ -0,0 +1,11 @@
+# Ussage of source{d} Community Edition
```

Comment:
`Ussage` -> `Usage`

Reply:
ouch. I do this mistake 50% I write `usage`. Thanks for discovering it! :D